internal/pkg/term/progress/ecs.go

```diff
@@ -180,6 +198,89 @@ func (c *rollingUpdateComponent) renderAlarms(out io.Writer) (numLines int, err
 	return renderComponents(out, components)
 }
 
+type stoppedTasksInfo struct {
+	ids    []string
+	reason string
+}
+
+func (c *rollingUpdateComponent) renderStoppedTasks(out io.Writer) (numLines int, err error) {
+	if len(c.stoppedTasks) == 0 {
+		return 0, nil
+	}
+	header := []string{"TaskId", "CurrentStatus", "DesiredStatus"}
+	var rows [][]string
+	title := fmt.Sprintf("Latest %d %s stopped reason", len(c.stoppedTasks), english.PluralWord(len(c.stoppedTasks), "task", "tasks"))
+	title = fmt.Sprintf("%s%s", color.DullRed.Sprintf("✘ "), color.Faint.Sprintf(title))
+	childComponents := []Renderer{
+		&singleLineComponent{}, // Add an empty line before rendering task stopped events.
+		&singleLineComponent{
+			Text:    title,
+			Padding: c.padding,
+		},
+	}
+
+	taskInfoSlice := make([]stoppedTasksInfo, 0, len(c.stoppedTasks))
```

Comment:
I did n't use `map` because I want to preserve the order of stopped reasons when displayed .
What I need to have a map
1. I can change this to `map[string][]string` 
2. I need to sort this map in a particular order based on `StoppingAt` time.

Do you want me to do this or just keep this as is.

Reply:
I think it'll be better from time complexity's perspective. And possibly will make the code slightly cleaner too?